workload/plugins/custom-plugin-select-host-and-boot.py

```diff
@@ -0,0 +1,33 @@
+from rally.plugins.openstack import scenario
```

Comment:
It's maybe better to keep all your files separated from the master.

I suggest you to create a scenario dedicated to your use case : see https://github.com/BeyondTheClouds/enos-scenarios

If you want you can push your configuration there : e.g the workload dir